connector-drawer/src/main/resources/application.properties

```diff
@@ -0,0 +1,37 @@
+notifications.connector.kafka.incoming.group-id=notifications-connector-drawer
+notifications.connector.kafka.incoming.topic=${mp.messaging.tocamel.topic}
+notifications.connector.kafka.outgoing.topic=${mp.messaging.fromcamel.topic}
+notifications.connector.name=drawer
+
+# The following value matches the default value of the `connectionsPerRoute` option from the Camel `http` component.
+notifications.connector.seda.concurrent-consumers=20
+notifications.connector.seda.enabled=true
+# The following value matches the default value of the `connectionsPerRoute` option from the Camel `http` component.
+notifications.connector.seda.queue-size=20
+
```

Comment:
I'm not sure we'll use SEDA in this connector since the output will be a Kafka message. Let's remove SEDA from this PR and maybe add it later.
```suggestion
```